assets/tests/components/mapPage/searchResultsByProperty.test.tsx

```diff
@@ -237,4 +318,40 @@ describe("searchResultsByProperty", () => {
       })
     ).toBeInTheDocument()
   })
+
+  test("When locations section has more matches, includes a 'Show more' button", () => {
```

Comment:
suggestion (non-blocking): consider adding a test for there not being a 'Show more' button when there are no more matches

Reply:
Added in 4a361ecd.